resources/teamcity/web/keyman-web-release.sh

```diff
@@ -56,31 +56,17 @@ function _zip_and_upload_artifacts() {
 
   cd "${KEYMAN_ROOT}/web"
 
-  local UPLOAD_DIR="build/upload/$KEYMAN_VERSION"
+  local UPLOAD_DIR="build/upload/${KEYMAN_VERSION}"
 
   ./ci.sh prepare:downloads.keyman.com
 
-  write_download_info "$UPLOAD_DIR" "keymanweb-$KEYMAN_VERSION.zip" KeymanWeb zip web
+  write_download_info "${UPLOAD_DIR}" "keymanweb-${KEYMAN_VERSION}.zip" KeymanWeb zip web
 
-  rsync_args=(
-    '-vrzltp'                                # verbose, recurse, zip, copy symlinks, preserve times, permissions
-    '--chmod=Dug=rwx,Do=rx,Fug=rw,Fo=r'      # map Windows security to host security
-    '--stats'                                # show statistics for log
-    "--rsync-path=${RSYNC_PATH}"             # path on remote server
-    "--rsh=${RSYNC_HOME}\ssh -i ${USERPROFILE}\.ssh\id_rsa -o UserKnownHostsFile=${USERPROFILE}\.ssh\known_hosts"                  # use ssh
-    "${KEYMAN_VERSION}"                          # upload the whole build folder
-    "${RSYNC_USER}@${RSYNC_HOST}:${RSYNC_ROOT}/web/${KEYMAN_TIER}/" # target server + path
+  (
+    cd build/upload
+    tc_rsync_upload "${KEYMAN_VERSION}" "web/${KEYMAN_TIER}"
   )
```

Comment:
Is there a reason this is put into a subshell apart from the `cd`?

Reply:
No. I put it in the subshell so that we don't have to `cd` back